lizmap/modules/lizmap/classes/lizmapRepository.class.php

```diff
@@ -169,14 +169,14 @@ public function update($data, $ini)
         return $modified;
     }
 
-    public function getProject($key)
+    public function getProject($key, $context, $services)
```

Comment:
these parameters:

* are not documented
* should be given to the constructor instead